Statistics, Exercise 1

Jim O'Regan

Question 1:

One sample population proportion:

$P \pm Z \times \sqrt{\frac{P(1-P}{n}}$

n = 300

Mean:

$63 \div 300 = 0.21$

$P = 0.21$

$Z = 1.96$ (Table value)

$0.21 \pm 1.96 \times \sqrt{\frac{0.21(1-0.21))}{300}}$

$=0.21 \pm 1.96 \times 0.02$

$= 0.21 \pm 0.04$

$=$ **0.17** to **0.25**

Question 2:

One sample population proportion:

$P \pm Z \times \sqrt{\frac{P(1-P}{n}}$

n = 4063

$Z = 1.96$ (Table value)

Mean:

$533 \div 4063 = 0.1312$

$P = 0.1312$

$0.1312 \pm 1.96 \times \sqrt{\frac{0.1312(1-0.312)}{4063}}$

$=0.1312 \pm 1.96 \times 0.0053$

$= 0.1312 \pm 0.0103$

$=$ **0.1209** to **0.1415**

Exercise 3

In [12]:
x1 = [9.5, 10.5, 9.0, 9.75, 10.0, 13, 10, 13.5, 10, 9.5, 10, 9.75]
x2 = [12.5, 9.5, 13.5, 13.75, 12.0, 13.75, 12.5, 9.5, 12.0, 13.5, 12.0, 12.0]

Two independent samples, difference between the population means, small samples:

$\bar{X_1} - \bar{X_2} \pm Z \sqrt{\frac{\sigma_p^2}{n_1} - \frac{\sigma_p^2}{n_2}}$

$\sigma_p^2 = \frac{(n_1-1)s_1^2 + (n_2-1)s_2^2}{n_1+n_2+2}$



$H_0: \mu_1 = \mu_2$

$H_1: \mu_1 \ne \mu_2$

$\bar{X_1} = 10.375$

$\bar{X_2} = 12.2083$

$\sigma_1 = 1.396$

$\sigma_2 = 1.449$

$n_1 = 12$

$n_2 = 12$

In [38]:
1.396 ** 2, 1.449 ** 2

(1.9488159999999997, 2.0996010000000003)

In [39]:
(1.396 ** 2) * 11, (1.449 ** 2) * 11

(21.436975999999998, 23.095611)

In [40]:
21.437 + 23.096

44.533

In [41]:
44.533 /22


2.024227272727273

In [42]:
2.0242 / 12

0.16868333333333332

In [43]:
0.1687 * 2

0.3374

In [44]:
math.sqrt(0.3374)

0.5808614292583042

In [45]:
0.5809 * 2.07

1.2024629999999998

In [46]:
-1.833 - 1.2025, -1.833 + 1.2025

(-3.0355, -0.6305000000000001)

$\bar{X_1} - \bar{X_2} \pm Z \sqrt{\frac{\sigma_p^2}{12} - \frac{\sigma_p^2}{12}}$

$\sigma_p^2 = \frac{(12-1)(1.396)^2 + (12-1)(1.449)^2}{12+12+2}$

$\sigma_p^2 = \frac{(11)1.9488 + (11)2.0996}{22}$

$\sigma_p^2 = \frac{21.437 + 23.096}{22}$

$\sigma_p^2 = \frac{44.533}{22}$

$\sigma_p^2 = 2.0242$


$Z = 2.07$ (table value)


$-1.833 \pm 2.07 \times \sqrt{\frac{2.0242}{12} + \frac{2.0242}{12}}$

$-1.833 \pm 2.07 \times \sqrt{0.1687 + 0.1687}$

$-1.833 \pm 2.07 \times \sqrt{0.3374}$

$-1.833 \pm 2.07 \times 0.5809$

$-1.833 \pm 1.2025$

$[-3.0355, -0.6305]$


The populations are not the same with 95% confidence.



In [37]:
math.sqrt(0.0126)

0.11224972160321824

In [33]:
10.375-12.2083

-1.8332999999999995

In [26]:
def std(x):
    mean = np.mean(x)
    
    sq_sum = lambda x: (x - mean) ** 2
    n = len(x)
    
    sq_sum = list(map(sq_sum, x))
    return math.sqrt(sum(sq_sum) / (n - 1))


def sem(x):
    s_sq = std(x) ** 2
    n = len(x)
    
    return math.sqrt(s_sq / n)
